include/deal.II/matrix_free/operators.h

```diff
@@ -872,20 +876,21 @@ namespace MatrixFreeOperators
     // lost
     VectorView<Number> view_src_in(src.local_size(), src.begin());
```

Comment:
Here is what I mean: You need to have access to a (raw) pointer of `src` to make this work. I do not think we can implement this for all vector classes. But we do not have to - we would simply require the user to provide us with an appropriate interface.